dataland-frontend/src/components/pages/UnsubscribeFromMailsPage.vue

```diff
@@ -0,0 +1,139 @@
+<template>
+  <TheHeader :showUserProfileDropdown="false" />
+
+  <TheContent class="container">
+    <main>
+      <div>
+        <div v-if="!isUnsubscribed" class="subscribed">
+          <h1>We are sorry you want to unsubscribe from our mailing list.</h1>
```

Comment:
```suggestion
          <h1>We are sorry, you want to unsubscribe from our mailing list.</h1>
```

Reply:
changed to "We are sorry that you wish to unsubscribe from our mailing list."